cmd/main.go

```diff
@@ -108,6 +101,9 @@ func main() {
 	ctrl.SetLogger(logger)
 	klog.SetLogger(logger)
 
+	metricsAddr := viper.GetString("metrics-bind-address")
```

Comment:
wonder if we should call 'IsSet' here instead of checking empty values downstream?

Reply:
ah never mind, we don't need that